src/Patient_Dashboard/VisitDate.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { format, isValid, parseISO } from 'date-fns';
+
+const VisitDate = ({ datetime }) => {
+  const date = parseISO(datetime);
+  
+  if (!isValid(date)) {
+    return <span className="visit-date">Invalid Date</span>;
+  }
+
+  const formattedDate = format(date, 'MMM dd, yyyy hh:mm a');
```

Comment:
Date formatting seems to be in multiple places in this PR? Shouldn't date format logic just be in some DateUtils?